recipes/Avocado-Toast.qmd

```diff
@@ -0,0 +1,24 @@
+---
+  title: "Template for new recipe"
```

Comment:
- [ ] Please remove the whitespace before `title`.
- [ ] Please add the recipe title